docs/README.md

```diff
@@ -0,0 +1,325 @@
+for formal submission of a self-contained specification.
+
+## Security
+
+What makes EVE a secure-by-design system is that it has been developed from the
+ground up with security in mind. EVE doesn't rely on 3rd party or add-on components
+to provide trust in the system, but rather offers a set of well-defined principles
+on which a trust model can be built.
+
+We have made an effort to provide users of EVE with a system that is both
+practically secure and can be deployed in a zero-touch fashion. In order to achieve
+our goals we put forward a [few guiding principles](SECURITY.md).
+
+## EVE Controller
+
+Each Edge Node with EVE running on it can participate in sophisticated
+[Edge Computing scenarious](https://www.zdnet.com/article/10-scenarios-where-edge-computing-can-bring-new-value/)
+ranging from simple [data collection](https://blog.equinix.com/blog/2017/11/29/how-iot-data-collection-and-aggregation-with-local-event-processing-work/)
+all the way to [Edge Analytics and AI](https://www.datanami.com/2019/02/04/exploring-artificial-intelligence-at-the-edge/).
+While EVE provides an incredible amount of functionality to support these use
+cases, it needs to be told exactly how to orchestrate these functional building blocks
+by a *Controller*. In other words, without Controller telling a fleet of Edge
+Nodes what to do, EVE by itself may be capable, but it practically useless.
+
+This is a fundamental point of EVE's design: an end user (lets call her Diane
+The DevOps) trying to implement an Edge Computing scenario never communicates
+directly with a running EVE instance. There are no user names, passwords or a
+concept of "logging in" when it comes to EVE. All interactions that Diane will
+have are going to be through the EVE Controller. Diane expresses her overall
+intent to a controller using a Controller API and it is then the job of a
+controller to build an execution plan for a fleet of Edge Nodes running EVE.
+When each instance of EVE contacts the controller using EVE API it will receive
+its own portion of the overall execution plan (note that EVE API communications
+are  always initiated by EVE, never by the Controller):
+
+![EVE Controller](pics/EVEController.png)
+
+A good way to think about this is that Controller is to EVE what Kubernetes is
+to Docker Engine.
+
+An API between EVE and its Controller is considered a [public API of Project EVE](../api).
+Refer to [API documentation](../api/API.md) for detailed information on API
+end points and message format.
+
+EVE design makes the trust between an EVE instance and its controller extremely
+important to the overall security of your Edge Nodes. An attack vector of
+impersonating an EVE controller or an EVE instance can be used for extremely
+malicious effects (like turning a [fleet of Edge Nodes into a botnet](https://www.cloudflare.com/learning/ddos/glossary/mirai-botnet/)).
+EVE takes great care of making sure that an EVE instance can alwats trust its
```

Comment:
"great care in making sure... can always trust" (type: "alwats")

Reply:
10x! This is super annoying since I didn't see it highlighted by my spell checker :-(